### PR TITLE
Pm 86 upgrade agp 8 4 2_CI

### PR DESCRIPTION
**Pull request by GeraltRiv**
I create new PR twice, because after changes to yml, it refuse to run all build checks.
I'm not sure if it is a bug, @a-szotyori maybe you know more about this?

Fix ci

### CODE REVIEW

File: .github/workflows/bank-api-library.check.yml [hunk 1]
```diff
@@ -145,6 +145,13 @@ jobs:
 
       - name: run android lint
         run: ./gradlew bank-api-library:library:lint
+        with:
+          script: >
+            -PtestClientId="gini-mobile-test"
+            -PtestClientSecret="${{ secrets.GINI_MOBILE_TEST_CLIENT_SECRET }}"
```
Comment (GeraltRiv) [inline .github/workflows/bank-api-library.check.yml:151]: @a-szotyori 
Without this parameters lint failed with new gradle plugin.
Is it ok to add this parameters like this?

Comment (a-szotyori) [reply .github/workflows/bank-api-library.check.yml:151]: Interesting why the lint gradle task runs "androidTest" tasks which we configured to depend on our "injectTestProperties" task 🤔 

Could you check why the linter runs "androidTest" tasks? If you can't find a way to make the linter independent from "androidTest", then we can keep your fix.

**Review by a-szotyori — COMMENTED** [review]
Thank you, Eugene! The workflows should relaunch by editing the PR description. Our workflows have a PR edited trigger.

File: .github/workflows/bank-api-library.check.yml (hunk 1 above)
Comment (GeraltRiv) [reply .github/workflows/bank-api-library.check.yml:151]: I can't find how to make linter independent. I think we need to create a task for this, and return to it later.

Comment (a-szotyori) [reply .github/workflows/bank-api-library.check.yml:151]: Thank you for checking it! Then we can keep your fix. 👍

**Review by a-szotyori — APPROVED** [review]
Thank you, looks great! Please don't merge yet. We need to do QA on this first. 

You can ask Daniela how to upload and share Bank SDK example app and Health SDK example app builds with AppCenter or I can brief you on Monday.

File: .github/workflows/bank-api-library.check.yml (hunk 1 above)
Comment (GeraltRiv) [reply .github/workflows/bank-api-library.check.yml:151]: @a-szotyori 
Ok, can you approve the PR?

**Comment by a-szotyori** [comment]
Hi, @GeraltRiv, I changed the target branch to `PM-86-android-15` to collect there all changes related Android 15.

Oh and please use the ticket id of the ticket itself (not its parent ticket) in the branch name. So this branch name should have used PM-85. Since this branch also contains changes for [PM-92](https://ginis.atlassian.net/browse/PM-92) the branch name then should contain both ids: `PM-85-PM-92-...`

Also I just noticed that the ticket id's in the comments are PM-68 and not PM-86. 🙈 

Please make the following changes:
- update the branch name to start with `PM-85-PM-92-...`.
- rewrite all commit messages to refer to the correct ticket ids.
- close this PR and open a new one for the new branch name and use `PM-86-android-15` as the merge target.

You can use rebase freely inside your feature branches to update the commit messages.

**Comment by a-szotyori** [comment]
Oh and also some additional `android lint` builds need those pesky test parameters 🙈

**Comment by a-szotyori** [comment]
Please ignore my last comment. I just saw your comment in the ticket.

**Pull request closed without merging**